core/trino-main/src/main/java/io/trino/operator/aggregation/TypedSet.java

```diff
@@ -53,8 +59,10 @@
 
     private final int initialElementBlockOffset;
     private final long initialElementBlockSizeInBytes;
-    // size is the number of elements added to the TypedSet (including null).
-    // It is equal to elementBlock.size() - initialElementBlockOffset
+    /**
```

Comment:
keep as ordinary comment, since it's a private member

(i know, it's subjective, but the comment already exists, so let's just keep it)